pkg/arvo/lib/chat-json.hoon

```diff
@@ -141,6 +134,23 @@
           [%end (numb end.upd)]
           [%envelopes [%a (turn envelopes.upd enve)]]
       ==
+    [*@t *^json]
+  ==
+::
+++  update-to-json
+  |=  upd=chat-update
+  =,  enjs:format
+  ^-  json
+  %+  frond  %chat-update
+  %-  pairs
+  :~
+    ?:  =(%message -.upd)
+      ?>  ?=(%message -.upd)
```

Comment:
Maybe I missed this before, but I'm curious: isn't just `?:  ?=(%message -.upd)` practically equivalent here? I don't recognize any TMI-sensitive situations in the code below.

Reply:
Yes, it is equivalent. I'll make a note to go modify all of the JSON marks with that expression rather than the ?>  expression. (it's used across a variety of marks that i've written)